pycbc/distributions/sky_location.py

```diff
@@ -22,6 +22,7 @@
 
 
 from scipy.spatial.transform import Rotation
+from mhealpy import HealpixMap
```

Comment:
I'd make it an optional dependency, otherwise I think it's fine. There is a small tool that can help with that if you use the import_optional function from libutils.